metis/spec/setup.js

```diff
@@ -0,0 +1,15 @@
+// suppresses a specific React warning (comment raf out to bring it back)
+import 'regenerator-runtime/runtime';
+import nock from 'nock';
+const raf = (global.requestAnimationFrame = (cb) => setTimeout(cb, 0));
+
+const Enzyme = require('enzyme');
+const EnzymeAdapter = require('enzyme-adapter-react-16');
+// Setup enzyme's react adapter
+Enzyme.configure({ adapter: new EnzymeAdapter() });
+
+global.fetch = require('node-fetch');
+nock.disableNetConnect();
+nock.emitter.on('no match', req => {
```

Comment:
Helpful for debug purposes; when a request is made in a test that does not match a stub, information on that request is logged so that you can quickly figure out which urls & payloads need stubbing.